indico/modules/auth/blueprint.py

```diff
@@ -34,3 +36,14 @@
 _bp.add_url_rule('/register/<provider>', 'register', RHRegister, methods=('GET', 'POST'))
 
 _bp.add_url_rule('/reset-password/', 'resetpass', RHResetPassword, methods=('GET', 'POST'))
+
+
+with _bp.add_prefixed_rules('/user/<int:user_id>', '/user'):
+    _bp.add_url_rule('/accounts/', 'accounts', RHUserAccounts, methods=('GET', 'POST'))
+    _bp.add_url_rule('/accounts/remove/<identity>', 'remove-account', RHUserAccountsRemove)
```

Comment:
We usually use underscores in the blueprint names (since they'd be autogenerated from function names when using the `@blueprint.route` decorator which cannot contain dashes).
Besides that, this should really be a POST request. Deleting something with GET is a no-go.

I'd also switch the last two segments, i.e. `/accounts/<identity>/remove` (more RESTful - using 'DELETE /accounts/<identity>' would be even better but then you'd have to switch to AJAX). Anyway, I guess non-AJAX is fine for this. You don't remove multiple accounts that often.